e2e/wdio/platform/tests/standard-list-item.e2e-spec.ts

```diff
@@ -0,0 +1,189 @@
+            checkElArrIsClickable(standardListPg.sMultiList);
+            checkElArrIsClickable(standardListPg.sMultiCheckbox);
+        });
+
+        it('should check selected item count is displayed in the toolbar', () => {
+            webDriver.scrollIntoView(standardListPg.sMultiCheckbox);
+            expect(webDriver.getText(standardListPg.sMultiToolbar)).toContain('0 : Items selected');
+            webDriver.click(standardListPg.sMultiCheckbox);
+            expect(webDriver.getAttributeByName(standardListPg.sMultiCheckbox, 'aria-checked')).toBe('true');
+            expect(webDriver.getText(standardListPg.sMultiToolbar)).toContain('1 : Items selected');
+        });
+    });
+
+    describe('Standard List Item (ByLine)- Inverted Secondary text types examples:', function() {
+        it('should check border and styles', () => {
+            checkAttributeValueTrue(standardListPg.sInvtAttr, StandardLinkData.byLineAltAttr);
+            expect(webDriver.getCSSPropertyByName(standardListPg.sInvtList, StandardLinkData.borderAttr).value)
+                .toBe(StandardLinkData.solidStyle);
+        });
+
+        it('should check content and interactions', () => {
+            checkElementDisplayed(standardListPg.sInvtAvatar);
+            checkElementDisplayed(standardListPg.sInvtList);
+            checkElArrIsClickable(standardListPg.sInvtList);
+            checkElementText(standardListPg.sInvtList);
+        });
+
+        it('should check secondary text types', () => {
+            const elCount = webDriver.getElementArrayLength(standardListPg.sInvtListItem);
+            for (let i = 0; elCount > i; i++) {
+                expect(webDriver.getAttributeByName(standardListPg.sInvtListItem, StandardLinkData.secondaryAttr, i))
+                    .not.toBe(null, '');
+                expect(webDriver.getAttributeByName(standardListPg.sInvtListItem, StandardLinkData.secondaryAttr, i))
+                    .toBe(StandardLinkData.secondaryTypes[i]);
+            }
+        });
+    });
+
+    describe('Check orientation', function() {
+        it('should check RTL and LTR orientation', () => {
+            const areas = webDriver.elementArray(standardListPg.exampleAreaContainersArr);
+            const switchers = webDriver.elementArray(standardListPg.rtlSwitcherArr);
+            for (let i = 0; i < areas.length; i++) {
+                switchers[i].click();
+                expect(webDriver.getAttributeByName(standardListPg.exampleAreaContainersArr, 'dir', i)).toBe('rtl');
+                expect(webDriver.getCSSPropertyByName(standardListPg.exampleAreaContainersArr, 'direction', i).value).toBe('rtl');
+                switchers[i].click();
+                expect(webDriver.getAttributeByName(standardListPg.exampleAreaContainersArr, 'dir', i)).toBe('ltr');
+                expect(webDriver.getCSSPropertyByName(standardListPg.exampleAreaContainersArr, 'direction', i).value).toBe('ltr');
```

Comment:
We have method in base-component to check the orientation.

standardListPg.checkRtlSwitch(standardListPg.rtlSwitcherArr, standardListPg.exampleAreaContainersArr)

Reply:
I guess it's not merged yet because I don't have it.